Puzzlepart/did
# Export hours in user menu

### Your checklist for this pull request

### Review checklist

### Description
Export hours in user menu.

![image](https://user-images.githubusercontent.com/7606007/95575057-58d41700-0a2e-11eb-8caf-1a186cdb857a.png)

![image](https://user-images.githubusercontent.com/7606007/95575072-5d98cb00-0a2e-11eb-9ca3-a81938d45f49.png)


### How to test
1. Check out locally with [gh](https://github.com/cli/cli) or use [didolemp.loca.lt](https://didolemp.loca.lt/)
2. Log in if you're not already logged in
3. Open user menu
4. Click export button
5. Test the different export types

### Related issues
Closes #589
